packages/@tailwindcss-upgrade/src/utils/memcpy.ts

```diff
@@ -0,0 +1,7 @@
+export function memcpy<T extends object, U extends object | null>(target: T, source: U): U {
```

Comment:
Really don't think this name is explanatory, especially now that it's a "global" thing. What about `replaceEntries`?

Reply:
I took this from the C++ world (https://en.cppreference.com/w/cpp/string/byte/memcpy)

It's basically `Object.assign(dst, src)` but also clears all values in `dst` that are not in `src`.

I'm open to suggestions for a better name, btu I feel like `replaceEntries` is less correct 🤔

Reply:
Yeah I see how mempcy makes sense but it doesn't feel JS-y at all haha. I don't know of a better name too, for me it does clear out the entries of the object and puts new ones there that's why I suggested this. maybe it's `replaceObject` or so haha. If we can't align on anything another idea is also to just inline this and don't try to abstract?